roles/epel/meta/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+galaxy_info:
+  description: Install EPEL repository
+  author: mierdin
+  company: StackStorm
+  license: Apache
+  min_ansible_version: 1.9
+  platforms:
+    - name: Ubuntu
+      versions:
+        - trusty
+        - precise
+    - name: EL
+      versions:
+        - 6
+        - 7
+    - name: CentOS
```

Comment:
Not needed.
Previous name: `EL` covers both RHEL and CentOS.

At least I couldn't find occurencies like that in any galaxy role.